web-www/src/components/BackgroundSection.tsx

```diff
@@ -196,33 +205,22 @@ const BackgroundSection = ({
     );
   }
 
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
+
   return (
-    <>
-      <Box display={{ xs: 'block', sm: 'none' }}>
-        <BackgroundImage
-          Tag="section"
-          className={clsx(className, classes.root)}
-          fluid={imageDataMobile ? imageDataMobile : imageData}
-          backgroundColor="transparent"
-        >
-          {linearGradient !== 'unset' && <div className={classes.backgroundGradient} />}
-          {textJSX}
-          <div className={classes.children}>{children}</div>
-        </BackgroundImage>
-      </Box>
-      <Box display={{ xs: 'none', sm: 'block' }}>
-        <BackgroundImage
-          Tag="section"
-          className={clsx(className, classes.root)}
-          fluid={imageData}
-          backgroundColor="transparent"
-        >
-          {linearGradient !== 'unset' && <div className={classes.backgroundGradient} />}
-          {textJSX}
-          <div className={classes.children}>{children}</div>
-        </BackgroundImage>
-      </Box>
-    </>
+    <BackgroundImage
+      Tag="section"
+      className={clsx(className, classes.root)}
+      fluid={imageDataMobile && isMobile ? imageDataMobile : imageData}
```

Comment:
majority of the deleted above was repeated code, aside from passing `imageDataMobile` - this just DRYs up the code a bit and removes the unnecessary `Box`es (most of the diff here is prettier formatting)